docs/experience/integrations.md

```diff
@@ -0,0 +1,207 @@
+title: Sematext Experience Integrations
+descriptions: Sematext integrations with ready to use monitoring agents and log shippers for infrastructure and container monitoring, log management and analytics, alerting, chatops, and more. Our Cloud and on-premises platform exposes the Elasticsearch API and syslog receivers with built in anomaly detection, data analysis and visualization tools and services
```

Comment:
If description matters we should update it to be more about User Experience / Real User Monitoring

Reply:
I've added a more suiting description.